src/CommandingOfficers/Commander.java

```diff
@@ -333,13 +333,14 @@ public void receiveBattleEvent(final BattleSummary summary)
     // Do nothing if we're not involved
     if( minion != null && enemy != null )
     {
-      double power = 0;
+      double power = 0; // value in funds of the charge we're getting
+      
       // Add up the funds value of the damage done to both participants.
       power += myHPLoss / minion.model.maxHP * minion.model.getCost();
       // The damage we deal is worth half as much as the damage we take, to help powers be a comeback mechanic.
       power += myHPDealt / enemy.model.maxHP * enemy.model.getCost() / 2;
-      power /= CHARGERATIO_FUNDS; // Turn funds into units of power
-      power += myHPDealt / CHARGERATIO_HP; // Add power based on HP damage dealt; rewards aggressiveness.
+      // Add power based on HP damage dealt; rewards aggressiveness.
+      power += myHPDealt * CHARGERATIO_HP;
```

Comment:
Why change division to multiplication? Was this an error before?

Reply:
It wasn't. I changed the metric of CO power charge from "discretes" (or stars) to funds.
The exchange rate then changed to 100 funds/HP instead of 90 HP/star

Reply:
If we are going back to using "ability power" as our measurement unit for now, should we change this back too?

Reply:
Nah, the math works out the same, and it's less complicated.